### ISSUE TITLE
Explanation regarding line about prefix

### ISSUE THREAD

**Issue by woctezuma** [minimaxir/gpt-2-simple#48]
I cannot figure out what [this line](https://github.com/minimaxir/gpt-2-simple/blob/873db52d79b380a79cdda0012208a105428e9306/gpt_2_simple/gpt_2.py#L375) is supposed to achieve.

```python
if prefix:
    gen_text = prefix[0] + gen_text
```

**Comment by woctezuma** [comment]
I assume the `prefix` is used to generate text with this line.

```python
        if not prefix:
            out = sess.run(output)
        else:
            out = sess.run(output, feed_dict={
                    context: batch_size * [context_tokens]
})
```

And then each generated text is missing the first character, as if:
- the first character of the prefix was the only input,
- the output text was generated a character at a time.

**Comment by woctezuma** [comment]
I wonder whether this would explain #5 if the model fails to generate the whole prefix.

In my experience, the issue, i.e. the prefix is missing characters, only occurs when the prefix is not `<|startoftext|>`, so it might be cases for which the GPT-2 model fails to predict the prefix from the first few characters and the encoded "context".

**Comment by woctezuma** [comment]
Maybe fixed by #69?
